fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsMemos.ts

```diff
@@ -32,7 +32,7 @@ export class FwsMemos {
       flightPhaseInhib: [],
       simVarIsActive: this.fws.usrStartRefueling,
       whichCodeToReturn: () => [0],
-      codesToReturn: ['000005001'],
+      codesToReturn: ['280000009', '280000010'],
```

Comment:
```suggestion
      codesToReturn: ['280000009', this.fws.oneEngineRunning.get() ? '280000011' : '280000010'],
```

REFUEL PNL DOOR OPEN exists as both amber and green variant (depending on whether one engine is started or not). Thus, also please change the memo entry in ``EcamMessages/index.tsx`` to ```'280000011': '\x1b<4mREFUEL PNL DOOR OPEN',```